tests/Integration/PrestaShopBundle/Translation/Provider/ExternalModuleLegacySystemProviderTest.php

```diff
@@ -61,51 +83,69 @@ protected function setUp()
         );
 
         $this->provider
-            ->setModuleName('some_module')
-            ->setDomain('ModulesSomeModule')
+            ->setModuleName(self::MODULE_NAME)
         ;
     }
 
-    protected function tearDown()
-    {
-        self::$kernel->shutdown();
-    }
-
-    public function testGetTranslationDomains()
```

Comment:
if have the feeling that this part is not covered at all by tests: am I wrong?

Reply:
You are right, but the right way to make an integration test for it would be:

- For `getFilters()`: Test a module that uses core translations.
- For `getTranslationDomains()`: Insert translations in database, then test that they are retrieved in the translated catalogue.

Otherwise it's just testing a getter.